ui/src/state/chat/chat.ts

```diff
@@ -900,7 +911,13 @@ export function useDmIsPending(ship: string) {
 
 const selMultiDms = (s: ChatState) => s.multiDms;
 export function useMultiDms() {
-  return useChatState(selMultiDms);
+  const dms = useChatState(selMultiDms);
+
+  useEffect(() => {
+    useChatState.getState().fetchMultiDms();
+  }, []);
```

Comment:
Confirming that multi DMs should be scried every time `NewDm` component is visited? Or is the initial subscription on application load sufficient?

Reply:
I didn't want to add another scry to our initial load and scrying every time someone visited new dm felt decent because it's pretty infrequent. We should probably revisit in another perf PR sometime soon.